chart/epinio-ui/Chart.yaml

```
@@ -15,4 +15,4 @@ name: epinio-ui
 sources:
 - https://github.com/epinio/ui
 type: application
-version: 0.3.2
+version: 0.6.0
```

The helm chart version should be
```suggestion
version: 0.4.0
```